src/pages/about/blog/self-enrollment-overhaul/index.mdx

```diff
@@ -0,0 +1,146 @@
+
+import acceptInvitationHomepage from "./accept_invitation_homepage.png";
+import inviteStudent from "./invite_student.png";
+import selfEnrollmentSettingsFull from "./self_enrollment_settings_full.png";
+import enterEnrollmentCodeHomepage from "./enter_enrollment_code_homepage.png";
+import studentDetailsPage from "./student_details_page.png";
+import scheduledPublish from "./scheduled_publish.png";
+import addExtension from "./add_extension.png";
+import releaseCover from "./release_cover.png";
+
+export const meta = {
+  title: "Overhauled enrollment management",
+  date: "2026-01-01",
+  author: "Peter Stenger",
+  tags: ["Release"],
+  introImage: releaseCover,
+};
+
+The new self-enrollment and publishing systems are here! They bring many improvements, including the ability to manage everything in the UI (no need to edit JSON) and fine-grained self-enrollment controls.
+
+<BlogCalloutBox variant="info">
+  **Key Takeaways:**
+
+1. Your existing course instances don't need any changes.
+2. Going forward, how students can access your course instance is changing.
+   You can either give them a link/code to enroll (see the [self-enrollment section](#self-enrollment)), or invite them yourself (see the [invitations section](#invitations)).
+   There is no longer an enrollment page for students to find courses.
+
+</BlogCalloutBox>
+
+Let's get into all the new features!
+
+### Enrollment management
+
+You should now use the "Students" tab to manage enrollments. You no longer need to manage enrollments or access in JSON. There are two
+main ways to grant students access to your course:
+
+1. Provide students with a self-enrollment link or code
+2. Manually invite students to your course
+
+TODO Students page image
+
+### Self-enrollment
+
+Self-enrollment allows students to enroll in a course instance as long as they have a code or link to join the course.
+You can further limit self-enrollment to specific institutions or by setting a cutoff date for self-enrollment.
+
+<BlogImage
+  src={selfEnrollmentSettingsFull}
+  alt="Overview of self-enrollment settings"
+  caption="The full self-enrollment settings panel with all available options"
+  full
+/>
+
+#### Self-enrollment codes
+
+If you want to allow students to input a code to join your course, you should check "Use enrollment code for self-enrollment".
+If they access any link to your course, they will be prompted to enter the code.
+
+<BlogImage
+  src={enterEnrollmentCodeHomepage}
+  alt="Enter enrollment code on homepage"
+  caption="Students can enter an enrollment code from the homepage to join a course instance"
+  full
+/>
+
+### Invitations
+
+You can also explicitly invite students to your course. These invitations will show up on the homepage for students to accept.
+If you want full control over everyone in your course, consider disabling self-enrollment completely.
+
+You can invite a student to your course instance by clicking the "Invite" button on the "Students" tab.
```

Comment:
I think we should maintain focus on explaining the features, not writing a user manual.
```suggestion
```